netns/netns.go

```diff
@@ -1,7 +1,9 @@
+//nolint
 package netns
 
 import (
 	"github.com/pkg/errors"
+	//nolint
```

Comment:
what was the lint here? shadow?

Reply:
I've been trying to fix a linter error and was testing if any location for `//nolint` would cause the windows-latest linter to pass. However, the linter still appears to fail so I'm not sure what is causing it.

Reply:
Fixed by adding +build linux